templates/frontend/objects/monograph_full.tpl

```diff
@@ -425,15 +509,44 @@
 									{capture assign="name"}{translate key="submission.versionIdentity" datePublished=$iPublication->getData('datePublished')|date_format:$dateFormatShort version=$iPublication->getData('version')}{/capture}
 									<li>
-										{if $iPublication->getId() === $publication->getId()}
-											{$name}
-										{elseif $iPublication->getId() === $currentPublication->getId()}
-											<a href="{url page="catalog" op="book" path=$monograph->getBestId()}">{$name}</a>
+										{if $isChapterRequest}
+                                        	{capture}{$versionCounter--}{/capture}
+											{if $iPublication->getId()|in_array:$chapterPublicationIds}
+                                                {capture}{$chapterCounter--}{/capture}
+												{if $chapterCounter === 0 && $versionCounter > 0}
+                                                    {capture assign="versionSuffix"}{translate key="submission.chapterCreated"}{/capture}
+												{else}
+                                                    {capture assign="versionSuffix"}{/capture}
+												{/if}
+                                                {if $iPublication->getId() === $publication->getId()}
+                                                    {$name}{$versionSuffix}
+                                                {elseif $iPublication->getId() === $currentPublication->getId()}
+													<a href="{url page="catalog" op="book" path=$monograph->getBestId()|to_array:"chapter":$chapter->getSourceChapterId()}">{$name}</a>{$versionSuffix}
+                                                {else}
+													<a href="{url page="catalog" op="book" path=$monograph->getBestId()|to_array:"version":$iPublication->getId():"chapter":$chapter->getSourceChapterId()}">{$name}</a>{$versionSuffix}
+                                                {/if}
+											{else}
+                                                {if $chapterCounter > 0}
+                                                    {translate key="submission.withoutChapter" name=$name}
+												{else}
+													{$name}
+                                                {/if}
+											{/if}
 										{else}
-											<a href="{url page="catalog" op="book" path=$monograph->getBestId()|to_array:"version":$iPublication->getId()}">{$name}</a>
+                                            {if $iPublication->getId() === $publication->getId()}
+                                                {$name}
+                                            {elseif $iPublication->getId() === $currentPublication->getId()}
+												<a href="{url page="catalog" op="book" path=$monograph->getBestId()}">{$name}</a>
+                                            {else}
+												<a href="{url page="catalog" op="book" path=$monograph->getBestId()|to_array:"version":$iPublication->getId()}">{$name}</a>
+                                            {/if}
```

Comment:
This deserves a bigger discussion, but I think I'd prefer to have completely different templates for chapters and monographs, rather than scattering the differences throughout the template in this way. I know that will result in some duplication, but in terms of HTML/CSS layout, that should be ok (ie - the page can use the same overall layout).

We might want to split some things (like author handling) into its own template, and have each `chapter.tpl` and `monograph_full.tpl` load that template (eg - `{include file="authors.tpl" authors=$authors}`). But I think it will be easier to read and maintain if they are separate pages. We also want to be able to tailor the chapter page a bit more -- for example, it should be clearer that we're looking at one chapter of a book.